src/Deprecated.jl

```diff
@@ -1061,3 +1062,169 @@ function addMsgFactors!(subfg::AbstractDFG,
   end
   return msgfcts
 end
+
+
+
+##==============================================================================
+## TODO IS THIS DEPRECATED #TODO
+##------------------------------------------------------------------------------
+
+## TODO IS THIS DEPRECATED - only called in other code that looks to be deprecated
+
+# FIXME figure out if this can be consolidated and deprecated
+function dwnPrepOutMsg( fg::AbstractDFG,
+  cliq::TreeClique,
+  dwnMsgs::Array{LikelihoodMessage,1},
+  d::Dict{Symbol, T},
+  logger=ConsoleLogger()) where T
```

Comment:
please add @warn("deprecated...") msgs.  Also I have been putting new stuff near the top of the file, just under "Deprecate after v0.16".

Reply:
Ok, Can do warning, wanted it under a separate heading to first confirm before adding it to a version. Seems we just like opisites. Me adding at the bottom and you at the top :-)

Reply:
its funny yeah

Reply:
sorry also don't want to get in your way, i was more thinking that whatever is on its way does not really matter.  The few things i have under special heading is at the top so we dont forget about it.

Reply:
ok updated